examples/nlp/token_classification/punctuation_capitalization.py

```diff
@@ -54,7 +54,7 @@
 parser.add_argument("--ignore_start_end", action='store_false')
 parser.add_argument("--ignore_extra_tokens", action='store_false')
 parser.add_argument("--none_label", default='O', type=str)
-parser.add_argument("--shuffle_data", action='store_true')
+parser.add_argument("--shuffle_data", default=True, type=bool)
```

Comment:
This will always resolve to True, even if you do --shuffle_data=False. To use boolean, please you action=....
Why do you want to shuffle the data by default, i thought we always had False as default previously

Reply:
^ @ekmb. This is true, your argument now effectively does nothing

Reply:
hey, just to let you know, action='store_true' works differently:
  - when you run command **with --shuffle_data** it sets the flag to True
  - when you run the command **without it**, the flag won't be set to True

Reply:
wouldn't this "parser.add_argument("--shuffle_data", default=True, type=bool)" mean a named argument has to be used?
E.g. --shuffle_data=True/False (with True) if not supplied

Reply:
No you cannot do this `python punctuation_capitalization.py --shuffle_data=False` will result in args.shuffle_data being True